Gemfile.lock

```diff
@@ -482,8 +484,6 @@ DEPENDENCIES
   jquery-ui-rails!
   json (>= 2.3.0)
   json_matchers
-  libv8-node (= 18.19.0.0)
-  mini_racer
```

Comment:
Doing this temporarily for my builds to succeed. Will investigate later (potentially move to importmaps or esbuild?)